google/auth/pluggable.py

```diff
@@ -141,12 +157,13 @@ def retrieve_subject_token(self, request):
             raise ValueError(
                 "Executables need to be explicitly allowed (set GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES to '1') to run."
             )
+        interactive_mode = os.environ.get("GOOGLE_EXTERNAL_ACCOUNT_INTERACTIVE") == "1"
```

Comment:
This still doesn't match with the design, library should be setting this environment var, not reading it.

Reply:
Why not allow the user to set as well? Similar to `GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES`